webpack_config/makeConfig.js

```diff
@@ -278,7 +282,7 @@ module.exports = function(opts = {}) {
   // ====================
   const output = {
     path: path.resolve(config.path.output, options.outputDir),
-    filename: options.isProduction ? '[name].[chunkhash:8].js' : '[name].js',
+    filename: options.isProduction ? `[name].${clientHash}.js` : '[name].js',
```

Comment:
This will apply clientHash to both client and vendor JS, so it will be the same name even if vendor changes (and client does not.)